src/gui/application.cpp

```diff
@@ -141,18 +141,32 @@ bool Application::configVersionMigration()
 
     // Did the client version change?
     // (The client version is adjusted further down)
-    bool versionChanged = configFile.clientVersionString() != MIRALL_VERSION_STRING;
+    const auto versionChanged = configFile.clientVersionString() != MIRALL_VERSION_STRING;
 
     // We want to message the user either for destructive changes,
     // or if we're ignoring something and the client version changed.
-    bool warningMessage = !deleteKeys.isEmpty() || (!ignoreKeys.isEmpty() && versionChanged);
+    const auto warningMessage = !deleteKeys.isEmpty() || (!ignoreKeys.isEmpty() && versionChanged);
```

Comment:
IMO with this variable name, one would expect this to be a warning string

I know this is the way the variable was originally names but a clearer name would make this more readable, i.e. `shouldDisplayWarningMessage`